zulipterminal/ui_tools/views.py

```diff
@@ -790,7 +790,11 @@ def __init__(self, controller: Any) -> None:
         super().__init__(self.log)
 
     def keypress(self, size: urwid_Size, key: str) -> str:
-        if is_command_key('GO_BACK', key) or is_command_key('HELP', key):
+        if is_command_key('SCROLL_UP', key):
```

Comment:
You may wish to include the other movement keys here too - ie. 'K'/'J' equivalents?